Launcher/app/containers/FileLoader.js

```diff
@@ -222,3 +274,23 @@ export default class FileLoader extends Component {
     );
   }
 }
+
+function mapStateToProps(state) {
```

Comment:
I feel like I might still prefer this stuff being in a different file but co-located with the component. Also I feel like they should both be stored within their own folder and expose an `index.js`. I can take a crack at organizing stuff this way if you like.

Reply:
I agree with you that component based files should be directed into their own folder system with an exposed `index.js` that handles the routing and store preferences.